spec/GraphQLOverHTTP.md

```diff
@@ -313,12 +324,18 @@ type (as indicated by the `Content-Type` header).
 If the client does not supply a `Content-Type` header with a POST request, the
 server SHOULD reject the request using the appropriate `4xx` status code.
 
+Note: This is left as a "should" to allow your server to choose a different
+behavior should it wish to.
+
 A server MAY support POST requests encoded with and/or accepting other media
 types.
 
 If a client does not know the media types the server supports then it SHOULD
 encode the request body in JSON (i.e. with `Content-Type: application/json`).
 
+Note: Request encoding with media type `application/json` is supported by every
+compliant _server_.
```

Comment:
minor: My opinion is that italicizing keywords should be used always or never.

Reply:
The italics actually serves to turn the word into a [spec-md reference](https://spec-md.com/#sec-Definition-References), in this case clarifying that "server" definitely refers to "GraphQL-over-HTTP compliant server" or whatever we defined it to be.

Reply:
(But I would be perfectly okay with every appropriate mention of client/server being a reference.)